core/src/main/java/com/snowgears/shop/hook/WorldGuardHook.java

```diff
@@ -66,35 +80,48 @@ public static boolean isShopAllowed(Plugin worldGuardPlugin, Player player, Loca
             RegionQuery query = WorldGuard.getInstance().getPlatform().getRegionContainer().createQuery();
 
             // Check if we are not in a region.
-            ApplicableRegionSet regions = query.getApplicableRegions(BukkitAdapter.adapt(loc));
-            if (regions.size() == 0) {
-                // If we have no regions here, then we need to check if we are requiring the `allow-shops` worldguard flag
-                if (Shop.getPlugin().hookWorldGuard()) {
-                    // Allow shops ONLY in regions with the shop flag set
-                    return false;
-                }
-                // If we are not requiring `allow-shops` and we are not inside a region, then allow shop creation
-                return true;
-            }
+            // ApplicableRegionSet regions = query.getApplicableRegions(BukkitAdapter.adapt(loc));
+            // System.out.println("Regions: " + regions.size() + "|" + regions.getRegions());
+
+            // if (regions.size() == 0) {
```

Comment:
Code should be removed if it is no longer needed instead of commenting it out to keep things clean.